Tests/ApodiniTests/PropertiesTests/BindingTests.swift

```diff
@@ -0,0 +1,106 @@
+//
+//  BindingTests.swift
+//  
+//
+//  Created by Max Obermeier on 24.02.21.
+//
+
+import Foundation
+
+@testable import Apodini
+import XCTest
+import XCTApodini
+import ApodiniREST
+
+final class BindingTests: ApodiniTests, EnvironmentAccessible {
+    struct Greeter: Handler {
+        @Binding var country: String?
+        
+        func handle() -> String {
+            country ?? "World"
+        }
+    }
+    
+    struct ReallyOptionalGreeter: Handler {
+        @Binding var country: String??
+        
+        func handle() -> String? {
+            if let country = self.country {
+                return country ?? "World"
+            } else {
+                return nil
+            }
+        }
+    }
+        
+    var featured: String? = "Italy"
+    
+    @PathParameter var selectedCountry: String
+    @Environment(\BindingTests.featured) var featuredCountry
+    
+    @ComponentBuilder
+    var testService: some Component {
+        Greeter(country: .constant(nil))
+        Group("default") {
+            Greeter(country: Binding<String?>(.constant("USA")))
```

Comment:
It would be great to use:
```suggestion
            Greeter(country: "USA")
```
here. We should investigate if we can even support string interpolation here but I think that just enabling the usage of String Literals if the type is a string would be great!
Similar things apply to bools, floats and so on

Reply:
I don't think this is possible. We cannot have to implementations for one protocol even with different associated type requirements and the compiler isn't smart enough to coerce from string literal to Binding<String> and then to Binding<String?>. Best we can get is `Binding<String?>("USA")`.

Reply:
For some reason `Binding<String?>("USA")` won't work, however `.constant("USA")` on its own does work. I also created a `.some("USA")` that does basically the same as `constant`, except the naming and typing is adapted to the special case of optional-wrapping.